modules/monitoring-default-monitoring-components.adoc

```diff
@@ -29,14 +29,18 @@ By default, the {product-title} {product-version} monitoring stack includes thes
 |Alertmanager
 |The Alertmanager service handles alerts received from Prometheus. Alertmanager is also responsible for sending the alerts to external notification systems.
 
-|`kube-state-metrics` agent
-|The `kube-state-metrics` exporter agent (KSM in the preceding diagram) converts Kubernetes objects to metrics that Prometheus can use.
+|kube-state-metrics agent
+|The kube-state-metrics exporter agent (KSM in the preceding diagram) converts Kubernetes objects to metrics that Prometheus can use.
 
-|`openshift-state-metrics` agent
-|The `openshift-state-metrics` exporter (OSM in the preceding diagram) expands upon `kube-state-metrics` by adding metrics for {product-title}-specific resources.
+|monitoring-plugin
+|The monitoring-plugin dynamic plugin component deploys the monitoring pages in the *Observe* section of the {product-title} web console. 
```

Comment:
```suggestion
|The monitoring-plugin dynamic plug-in component deploys the monitoring pages in the *Observe* section of the {product-title} web console. 
```
Per the IBMSG, it should be "plug-in": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#plug-in

Reply:
Thanks @abrennan89. I think that [the SSG guidance to use "plugin" rather than "plug-in"](https://redhat-documentation.github.io/supplementary-style-guide/#plugin-adj) overrides the SSG guidance. 

Wouldn't it be nice if we had all of our style guidelines in a single place?